ksql-execution/src/main/java/io/confluent/ksql/execution/materialization/MaterializationInfo.java

```diff
@@ -133,6 +145,15 @@ public Builder filter(Expression filterExpression) {
     public MaterializationInfo build() {
       return new MaterializationInfo(stateStoreName, stateStoreSchema, transforms, schema);
     }
+
+    /*
+     * Materialized tables do not have meta columns, such as ROWTIME, as this would be obtained
+     * from the source event triggering the output in push / persistent query. Materialized tables
```

Comment:
see my previous comment, I don't think this explanation is convincing.

Reply:
see https://github.com/confluentinc/ksql/issues/3623